src/components/App/App.css

```diff
@@ -1,21 +1,113 @@
 .App {
   text-align: center;
   background-repeat: repeat;
-  height: 100vh;
   display: flex;
   flex-direction: column;
-  background-image: url("https://images.unsplash.com/photo-1573209946145-848669b5ef39?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1134&q=80");
-  background-repeat: no-repeat;
-  background-size: cover;
+  font-family: 'Arvo', serif;
+}
+
+.header {
+  background-color: #EFEFEF;
 }
 
 h1 {
   font-size: 2rem;
   text-transform: uppercase;
   text-align: left;
   margin: 0.75rem 1rem;
+  font-family: 'Lilita One', cursive;
 }
 
 .active {
   border-bottom: 3px solid black;
 }
+
+.call-to-action-question {
+  margin: 0 1rem;
+  font-size: 1.8rem;
+}
+
+.fact-container {
+  color: #3a526a;
+  padding: 1rem;
+  font-size: 1.3rem;
+}
+
+.arrows-container {
+  display: flex;
+  justify-content: space-between;
+}
+
+.arrow-button {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  border-radius: 50%;
+  border: 0;
+  height: 5vh;
+  cursor: pointer;
+}
+
+.did-you-know {
+  text-transform: uppercase;
+  font-family: 'Lilita One', cursive;
+  font-size: 1.8rem;
+}
+
+.quiz-call-to-action {
+  background-image: url("https://images.unsplash.com/photo-1573209946145-848669b5ef39?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1134&q=80");
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 60vh;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-around;
+  align-items: center;
+}
+
+.retake-button,
```

Comment:
You've got cleanly-written CSS here, and way these are grouped make sense to me (even as someone not familiar with the code).  As a stretch goal, you could always use SCSS as well to further bundle the code.